tools/database/src/analyze_gas_usage.rs

```diff
@@ -242,7 +242,8 @@ fn get_gas_usage_in_block(
             let account_shard_id = shard_layout.account_id_to_shard_id(&outcome.executor_id);
             assert_eq!(account_shard_id, shard_id);
 
-            gas_usage_in_shard.add_used_gas(outcome.executor_id, outcome.gas_burnt.into());
+            gas_usage_in_shard
+                .add_used_gas(outcome.executor_id, outcome.gas_burnt.as_gas() as u128);
```

Comment:
You can keep `.into()` for u64->u128 convertions, and this will be type-safe without a risk of silently turn i64->u128 if something gets changed:

```suggestion
                .add_used_gas(outcome.executor_id, outcome.gas_burnt.as_gas().into());
```